qsirecon/workflows/recon/converters.py

```diff
@@ -154,6 +155,7 @@ def init_qsirecon_to_fsl_wf(
             DerivativesDataSink(
                 dismiss_entities=("desc",),
                 suffix="mask",
+                extension=".nii.gz",
```

Comment:
Does this need `compress=True`? Or does the extension take care of that?

Reply:
Either works. `compress=True` is probably more flexible if we ever support non-NIfTI outputs here.